src/main/java/marquez/api/GetAllNamespacesResponse.java

```diff
@@ -0,0 +1,25 @@
+package marquez.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import marquez.core.models.Namespace;
+
+public class GetAllNamespacesResponse {
+
+  List<GetNamespaceResponse> namespaces = new ArrayList<>();
```

Comment:
Wouldn't `GetAllNamespacesResponse` contain a list of namespaces, not a list of `GetNamespaceResponse`. You can view `XResponse` or `XRequest` as strictly the HTTP body.

Reply:
Oh, I see why you used `GetNamespaceResponse`, there are fields we don't want to return. We still wouldn't use `GetNamespaceResponse` given it's a bit awkward, wondering if the naming convention `XResponse` / `XRequest` adds much benefits.

Reply:
Ok so removing the HTTP verb from the name? That might work in the case that the response for a PUT and a GET would be the same. However, looking at the api spec, that's not true in some cases (e.g. jobRun PUT vs. jobRun GET).

Reply:
I'm totally on-board that the naming/semantics are a little funky. Open to suggestions.

Reply:
Fixed in latest version.